Auth0/Authentication.swift

```diff
@@ -1114,29 +1126,33 @@ public extension Authentication {
                challenge: PasskeyLoginChallenge,
                connection: String? = nil,
                audience: String? = nil,
+               organization: String? = nil,
                scope: String = defaultScope) -> Request<Credentials, AuthenticationError> {
         return self.login(passkey: passkey,
                           challenge: challenge,
                           connection: connection,
                           audience: audience,
+                          organization: organization,
                           scope: scope)
     }
 
     @available(iOS 16.6, macOS 13.5, visionOS 1.0, *)
-    func passkeyLoginChallenge(connection: String? = nil) -> Request<PasskeyLoginChallenge, AuthenticationError> {
-        return self.passkeyLoginChallenge(connection: connection)
+    func passkeyLoginChallenge(connection: String? = nil, organization: String? = nil) -> Request<PasskeyLoginChallenge, AuthenticationError> {
+        return self.passkeyLoginChallenge(connection: connection, organization: organization)
     }
 
     @available(iOS 16.6, macOS 13.5, visionOS 1.0, *)
     func login(passkey: SignupPasskey,
                challenge: PasskeySignupChallenge,
                connection: String? = nil,
                audience: String? = nil,
+               organization: String? = nil,
                scope: String = defaultScope) -> Request<Credentials, AuthenticationError> {
         return self.login(passkey: passkey,
                           challenge: challenge,
```

Comment:
There is a warning about infinite recursion here. Were all the functions tested manually?